### PR TITLE
feat: Implement Glue registries resource

### LINKED ISSUE #1324: AWS::Glue::Registry resource
Implement AWS::Glue::Registry resource

### PR DESCRIPTION
**Pull request by irmatov**
Closes #1324

### CODE REVIEW

File: resources/services/glue/registries.go [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+func fetchGlueRegistrySchemas(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	r := parent.Item.(types.RegistryListItem)
+	cl := meta.(*client.Client)
+	svc := cl.Services().Glue
+	input := glue.ListSchemasInput{
+		RegistryId: &types.RegistryId{RegistryArn: r.RegistryArn},
+		MaxResults: aws.Int32(100),
+	}
+	for {
+		result, err := svc.ListSchemas(ctx, &input)
```
Comment (amanenk) [inline resources/services/glue/registries.go:209]: There is an automated way of `list - get` process. it should be used here. Look at `client.ListAndDetailResolver`

**Review by amanenk — APPROVED** [review]
LGTM. the code can be standardized a bit.

**Comment by hermanschaaf** [comment]
@irmatov If you bring this branch up to date with the main branch, the integration tests should no longer be a requirement and I'd expect the build to pass.

File: resources/services/glue/registries.go [hunk 2]
```diff
@@ -340,13 +337,7 @@ func resolveGlueRegistrySchemaVersionMetadata(ctx context.Context, meta schema.C
 	svc := cl.Services().Glue
 	s := resource.Item.(*glue.GetSchemaVersionOutput)
 	input := &glue.QuerySchemaVersionMetadataInput{
-		SchemaId: &types.SchemaId{
-			SchemaArn: s.SchemaArn,
-		},
 		SchemaVersionId: s.SchemaVersionId,
-		SchemaVersionNumber: &types.SchemaVersionNumber{
-			VersionNumber: s.VersionNumber,
```
Comment (hermanschaaf) [inline resources/services/glue/registries.go]: @amanenk Are the `SchemaId` and `SchemaVersionNumber` params not necessary? What version number and Id does it use then?

Comment (amanenk) [reply resources/services/glue/registries.go]: During testing with real deployment I got bad request errors because of redundant request params.

**Review by hermanschaaf — APPROVED** [review]

File: docs/tables/aws_glue_registry_schema_versions.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
+# Table: aws_glue_registry_schema_versions
+
```
Comment (candiduslynx) [inline docs/tables/aws_glue_registry_schema_versions.md:3]: description?

File: docs/tables/aws_glue_registry_schemas.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+
+# Table: aws_glue_registry_schemas
+
```
Comment (candiduslynx) [inline docs/tables/aws_glue_registry_schemas.md:3]: description?

**Review by candiduslynx — CHANGES_REQUESTED** [review]

**Review by bbernays — APPROVED** [review]

**Review by candiduslynx — APPROVED** [review]

**Pull request merged**